e2e-tests/portal/cypress/e2e/journeys/payment-officer/utilisation-report-upload/confirm-and-send.spec.js

```diff
@@ -0,0 +1,36 @@
+const { utilisationReportUpload, confirmAndSend } = require('../../../pages');
+const MOCK_USERS = require('../../../../fixtures/users');
+const relativeURL = require('../../../relativeURL');
+
+const { BANK1_PAYMENT_OFFICER1 } = MOCK_USERS;
+
+context('Confirm and send', () => {
+  beforeEach(() => {
+    cy.login(BANK1_PAYMENT_OFFICER1);
+    cy.visit(relativeURL('/utilisation-report-upload'));
+
+    utilisationReportUpload.utilisationReportFileInput().attachFile('test-csv.csv');
```

Comment:
Note that when my work is added we'll need to use the 'valid utilisation report' named file (probably my job as this will get merged first!)